src/pydolphinscheduler/core/task.py

```diff
@@ -144,6 +145,8 @@ def __init__(
         condition_result: Optional[Dict] = None,
         resource_plugin: Optional[ResourcePlugin] = None,
         is_cache: Optional[bool] = False,
+        input_params: Optional[Dict] = None,
+        output_params: Optional[Dict] = None,
```

Comment:
BTW, please add doctoring for this two new parameter

Reply:
Done